src/service/impl/stops/departure-time.ts

```diff
@@ -51,13 +45,12 @@ export async function populateCacheIfNotThere(
 }
 
 export async function getRealtimeDepartureTime(
-  inputQuery: DepartureRealtimeQuery,
-  client: GraphQLClient = journeyPlannerClient
+  inputQuery: DepartureRealtimeQuery
 ): Promise<Result<DeparturesRealtimeData, APIError>> {
   try {
     const variables = createVariables(inputQuery);
-    const previousResult = getPreviousExpectedFromCache(variables, client);
-    const result = await client.query<
+    const previousResult = getPreviousExpectedFromCache(variables);
+    const result = await journeyPlannerClient_v3.query<
       GetDepartureRealtimeQuery,
       GetDepartureRealtimeQueryVariables
```

Comment:
@gorandalum 
> Could you elaborate some on "Removed the option to get realtime with OTP1."?

Ah, I realize that's not a great explanation. It essentially means I removed the `client` param here from `getRealtimeDepartureTime` and `getPreviousExpectedFromCache`, which was used to choose between OTP versions.